pom.xml

```diff
@@ -11,7 +11,7 @@
 	</parent>
 
 	<artifactId>matlab</artifactId>
-	<version>2.15.1-SNAPSHOT</version>
+	<version>2.16.0-SNAPSHOT</version>
```

Comment:
Why is the version updated? Generally, the version is updated during the maven release process.